sky/packages/sky/lib/src/widgets/input.dart

```diff
@@ -22,40 +22,42 @@ class Input extends StatefulComponent {
 
   Input({
     GlobalKey key,
+    this.value : '',
     this.placeholder,
     this.onChanged,
     this.keyboardType : KeyboardType_TEXT
   }): super(key: key);
 
   int keyboardType;
+  String value;
   String placeholder;
   StringValueChanged onChanged;
 
-  String _value = '';
   EditableString _editableValue;
   KeyboardHandle _keyboardHandle = KeyboardHandle.unattached;
 
   void initState() {
     _editableValue = new EditableString(
-      text: _value,
+      text: value,
       onUpdated: _handleTextUpdated
     );
     super.initState();
   }
 
   void syncConstructorArguments(Input source) {
+    value = source.value;
```

Comment:
I've updated the code, but I have trouble understanding the motivation.
We allow the user of the API to update the placeholder text, correct?  Why not also allow to update the string value of the component?  That is arguably more useful.